scalardb-test/write-skew-config.toml

```
@@ -0,0 +1,41 @@
+[modules]
+  [modules.preprocessor]
+    name = "kelpie.scalardb.transfer.TransferPreparer"
+    path = "build/libs/scalardb-test-all.jar"
+  [modules.processor]
+    name = "kelpie.scalardb.transfer.WriteSkewProcessor"
```

```suggestion
    name = "kelpie.scalardb.transfer.WriteSkewTransferProcessor"
```
because it is still doing transfer but it is writing skewly.